test/unit/spec/MfaVerify_spec.js

```diff
@@ -1994,6 +2012,24 @@ function (Okta,
               });
             });
           });
+          itp('calls authClient verifyFactor with correct args for autoPush when submitted', function () {
+            return setupOktaPush({'features.autoPush': true}).then(function (test) {
+              $.ajax.calls.reset();
+              setAutoPushCheckbox(test, true);
```

Comment:
do we have one test for when autoPush is not checked?

Reply:
No, i didn't add one because it is covered in auth-js. I'll add them now.